app/static/src/app/components/sessions/SessionsPage.vue

```diff
@@ -283,9 +294,9 @@ export default defineComponent({
             window.location.assign(link);
         };
 
-        onMounted(() => {
+        onMounted(async () => {
+            await store.dispatch(AppStateAction.LoadUserPreferences);
```

Comment:
We need to load user preferences now before fetching sessions metadata as we use the duplicates preference in query string parameter to the endpoint.